game-app/game-core/src/test/java/games/strategy/triplea/ui/panels/map/MapRouteDrawerTest.java

```diff
@@ -37,6 +37,13 @@
 import org.triplea.java.collections.IntegerMap;
 
 final class MapRouteDrawerTest {
+  /**
+   * This is a very special value because {@code 13.44 + Double.MIN_NORMAL == 13.44} evaluates to
+   * true. So we use the square of this value (because the code computes the square root) in order
```

Comment:
Describing how/why this is a specific edge case would be useful. As a read I'm not sure if that commentary adds much value as I wonder about the edge case (what it is, what it effects). In combination with this, renaming 'Magic value' to something more descriptive would be a related change to generally be more descriptive.

Perhaps the comment could contrast the non-edge case value, in other words, why is `13.44 + Double.MIN_NORMAL` surprising, what would we expect it to equal?

Reply:
Good point. Without the old code this comment does not make a lot of sense. I will try adding some context to this.

Regarding the variable name:
I must admit that the variable name was chosen mainly for 2 reasons.
1. I wasn't able to come up with a rather short and still descriptive name (that's why I added the javadoc in the first place)
2. MAGIC_VALUE is still short enough so we don't need a line break in the method source method below 😅 

Still not sure how to call it, if you have a good Idead, please let me know.

Reply:
It's hard to suggest a value without knowing more about why it is a magic value. I think it's worth bearing in mind that the standards for a clean test is different from production code. A named constant could get out of hand if we did that for a half a dozen interesting values. I'd recommend to instead elide this problem, we only use the value on one line. Instead inline it and add a line or two of commentary above the `Arguments.of` line explaining why `13.44` is so particularly interesting.

Reply:
Sounds like a good idea 👍🏼